### DISCUSSION TITLE
Compiler option -fdump-ipa-clones is unsupported

### DISCUSSION THREAD

**Discussion by agustinhenze** [ccache/ccache#383] [Support]
Hi, I'm building a rhel kernel and i'm getting the message mentioned in the subject. Not sure if opening an issue here is ok. I was wondering if the only way to avoid this is removing that compiler option?

**Comment by agustinhenze** [comment]
OK, I can see there is an explicit if in the code for this :thinking: 
```C
		// These are always too hard.
		if (compopt_too_hard(argv[i]) || str_startswith(argv[i], "-fdump-")) {
			cc_log("Compiler option %s is unsupported", argv[i]);
			stats_update(STATS_UNSUPPORTED_OPTION);
			result = false;
			goto out;
		}

```

**Comment by jrosdahl** [comment]
Hi,

> I was wondering if the only way to avoid this is removing that compiler option?

No, there is no other way than avoiding that compiler option or implementing support for it in ccache. I had never heard of it, but it looks like it creates a (or potentially several?) extra files alongside the object file, so support for it won't happen until #218 is implemented.

**Comment by Werkov** [comment]
FWIW, see #1449.
